src/azure-cli-core/azure/cli/core/_output.py

```diff
@@ -131,11 +131,17 @@ def _capitalize_first_char(x):
     @staticmethod
     def _auto_table_item(item):
         new_entry = OrderedDict()
-        for k in item.keys():
-            if k in TableOutput.SKIP_KEYS:
-                continue
-            if item[k] and not isinstance(item[k], (list, dict, set)):
-                new_entry[TableOutput._capitalize_first_char(k)] = item[k]
+        try:
+            for k in item.keys():
+                if k in TableOutput.SKIP_KEYS:
+                    continue
+                if item[k] and not isinstance(item[k], (list, dict, set)):
+                    new_entry[TableOutput._capitalize_first_char(k)] = item[k]
+        except AttributeError:
+            # handles odd cases where a string/bool/etc. is returned
+            if isinstance(item, bool):
+                item = 'Success' if item else 'Fail'
```

Comment:
Well, the lovely thing about storage is that it will actually never return "Fail". It will return "Success" (or True) or nothing at all. I chose "Success" because, other than the exists call, "Success" makes more sense in context for most storage commands than "True". For example, if you create a blob, "Success" is a more logical response than "True". One of the many many things I love about storage.